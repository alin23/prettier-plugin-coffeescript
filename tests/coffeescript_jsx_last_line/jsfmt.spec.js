run_spec(__dirname, ['coffeescript'], { jsxBracketSameLine: true })
run_spec(__dirname, ['coffeescript'], { jsxBracketSameLine: false })
