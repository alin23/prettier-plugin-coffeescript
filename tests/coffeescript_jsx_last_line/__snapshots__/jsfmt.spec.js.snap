// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`last_line.coffee 1`] = `
<SomeHighlyConfiguredComponent
  onEnter={this.onEnter}
  onLeave={this.onLeave}
  onChange={this.onChange}
  initialValue={this.state.initialValue}
  ignoreStuff={true}
>
  <div>and the children go here</div>
  <div>and here too</div>
</SomeHighlyConfiguredComponent>
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
<SomeHighlyConfiguredComponent
  onEnter={@onEnter}
  onLeave={@onLeave}
  onChange={@onChange}
  initialValue={@state.initialValue}
  ignoreStuff={true}>
  <div>and the children go here</div>
  <div>and here too</div>
</SomeHighlyConfiguredComponent>

`;

exports[`last_line.coffee 2`] = `
<SomeHighlyConfiguredComponent
  onEnter={this.onEnter}
  onLeave={this.onLeave}
  onChange={this.onChange}
  initialValue={this.state.initialValue}
  ignoreStuff={true}
>
  <div>and the children go here</div>
  <div>and here too</div>
</SomeHighlyConfiguredComponent>
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
<SomeHighlyConfiguredComponent
  onEnter={@onEnter}
  onLeave={@onLeave}
  onChange={@onChange}
  initialValue={@state.initialValue}
  ignoreStuff={true}
>
  <div>and the children go here</div>
  <div>and here too</div>
</SomeHighlyConfiguredComponent>

`;
